projects/go-lib/src/lib/components/go-button/go-button.component.ts

```diff
@@ -18,22 +17,28 @@ export class GoButtonComponent {
 
   constructor() {}
 
-  public clicked() : void {
-    if (this.isProcessing || this.buttonDisabled) { return; }
```

Comment:
We don't actually need this protection if we utilize the disabled attribute properly.